### DISCUSSION TITLE
Welcome to Cronica Discussions!

### DISCUSSION THREAD

**Discussion by MadeiraAlexandre** [MadeiraAlexandre/Cronica#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by alexanderfdeveloper** [comment]
O aplicativo está ficando mais maduro 😁, muitas coisas que eram confusas e deixavam a experiência ruim, estão sendo melhorada ![image](https://github.com/MadeiraAlexandre/Cronica/assets/94754185/3ced2439-1b0e-49d7-8b8f-2a9ba731754f)

Por exemplo, abro o app na tela inicial tem os episódios que estou assistindo, e posso marcar como assistido ao abrir e marcar como assistido e já mostra o próximo, (vou dizer que cliquei 2 vezes achando que também funciona ali kk)

Hoje abri e vi os filmes em alta, antes mal olhava a tela inicial, mas estavam comentando sobre 2 filmes e vi eles ao abrir o App, já li a sinopse e tbm compartilhar pelo botão, foi um experiência incrível ☺️

**Reply by MadeiraAlexandre** [reply]
Eu curti a ideia de um toque duplo marcar o episódio como assistido no 'A Seguir', vou ver se consigo implementar sem que entre em conflito com abrir os detalhes do episódio.
Fico muito feliz com seu comentário sobre o Cronica! Muito obrigado! 😁

**Comment by alexanderfdeveloper** [comment]
![image](https://github.com/MadeiraAlexandre/Cronica/assets/94754185/4352cff4-0513-4e77-8737-2b6c772841cf)

A nova melhoria no gerenciamento e sincronização de lista com o tmdb, está ficando incrível 😁

![image](https://github.com/MadeiraAlexandre/Cronica/assets/94754185/6fab0ef5-bc19-45bf-bfed-c02ce37b4f6e)

**Reply by MadeiraAlexandre** [reply]
Planejo adicionar uma opção na criação de lista para permitir criar uma lista no TMDB, além de uma melhor edição delas. 
Se tiver sugestões ou ver algo que não tá funcionando bem, pode ir mandando nos Issues que eu estou de olho pra melhorar a experiência com o app! 😁
Obrigado pela ajuda e pelo feedback sobre a mudança!

**Comment by alexanderfdeveloper** [comment]
Apesar do conteúdo ser vasto, ainda pode faltar alguns e outros não estarem traduzido, então conforme vou fazer um cópia do mydramalist para o TMDB, tbm vou adicionando ou atualizando

Faz um tempo que queria aprender Swift, mas no momento já estou atolado kkk

Acharia interessante de ajudar e adicionar recursos de edição, porque vi que tem endpoint pra isso 😁

![image](https://github.com/MadeiraAlexandre/Cronica/assets/94754185/d725ba95-a9ef-4c9c-a021-1c9c08f97364)
